src/ducks/sep24DepositAsset.ts

```diff
@@ -126,11 +127,11 @@ export const depositAssetAction = createAsyncThunk<
     } catch (error) {
       log.error({
         title: "Deposit failed",
-        body: error.message,
+        body: getErrorMessage(error),
       });
 
       return rejectWithValue({
-        errorString: error.message,
+        errorString: getErrorMessage(error),
```

Comment:
same as above - probably best to cache error when you're logging and then rejecting with the same string